libs/simdvec/src/main21/java/org/elasticsearch/simdvec/VectorScorerFactoryImpl.java

```diff
@@ -57,7 +57,7 @@ public Optional<RandomVectorScorerSupplier> getInt7SQVectorScorerSupplier(
     @Override
     public Optional<RandomVectorScorer> getInt7SQVectorScorer(
         VectorSimilarityFunction sim,
-        RandomAccessQuantizedByteVectorValues values,
+        QuantizedByteVectorValues values,
```

Comment:
for my info, how do you all make changes to the java files under main21 and main22 from your IDE? They don't look like source dirs in my project.

Reply:
When I import the Gradle project, then the source sets are added automatically.

Reply:
interesting! I shall reimport the project then.